### ISSUE TITLE
Reader: add focus styles for navigating content with a keyboard

### ISSUE THREAD

**Issue by MichaelArestad** [Automattic/wp-calypso#21889]
Currently, there are no focus styles in Reader. This means it's kind of a mystery as to what article or link you have targeted. Let's get this fixed!

**Comment by jancavan** [comment]
Just a note: I was looking into this, but it doesn't seem to be an issue specific to the Reader. Filed an issue here: https://github.com/Automattic/wp-calypso/issues/23102

We do, however, need to update the `:focus` styles to use the blue border and have the link colors change like in other areas of Calypso.

**Comment by MichaelArestad** [comment]
We investigated this a bit more last week and it appears the focus styles don't work in Firefox.
